clients/src/main/java/org/apache/kafka/common/requests/FindCoordinatorResponse.java

```diff
@@ -45,12 +43,12 @@
     private final Errors error;
     private final Node node;
 
-    public GroupCoordinatorResponse(Errors error, Node node) {
+    public FindCoordinatorResponse(Errors error, Node node) {
```

Comment:
Can I do this in a follow up PR? Seems like a bigger refactor given the existing usage of the `FindCoordinatorResponse`. Also, there are other PRs waiting behind this one, so would prefer to close this sooner rather than later.